tests/test_elements.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+
+from ansys.dpf import core as dpf
+
+
+@pytest.fixture()
+def model_elements(allkindofcomplexity):
+    model = dpf.Model(allkindofcomplexity)
+    elements = model.metadata.meshed_region.elements
+    return elements
+
+@pytest.fixture()
+def hexa_element_descriptor(model_elements):
+    el_index = model_elements.scoping.index(251)
+    descriptor = dpf.element_types.descriptor(model_elements[el_index].type)
+    return descriptor
+
+@pytest.fixture()
+def quad_element_descriptor(model_elements):
+    el_index = model_elements.scoping.index(10660)
+    return dpf.element_types.descriptor(model_elements[el_index].type)
+
+@pytest.fixture()
+def tetra_element_descriptor(model_elements):
+    el_index = model_elements.scoping.index(3905)
+    return dpf.element_types.descriptor(model_elements[el_index].type)
+
+@pytest.fixture()
+def line_element_descriptor(model_elements):
+    el_index = model_elements.scoping.index(10834)
+    return dpf.element_types.descriptor(model_elements[el_index].type)
+
+def test_hexa_element_descriptor(hexa_element_descriptor):
```

Comment:
You may consider using [Parameterized test functions](https://docs.pytest.org/en/6.2.x/parametrize.html) rather than writing out several tests.